.github/workflows/knquickstart.yaml

```diff
@@ -42,6 +42,12 @@ jobs:
 
     steps:
 
+    # install Go locally
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
```

Comment:
go-version requirements seem different. Is this intentional?

Reply:
Yes. It was my thought that we can get more coverage by testing different versions of go in e2etests and knquickstart.  I can make them the same if you think this not worth doing.  Thoughts?